### DISCUSSION TITLE
Is there an SDK event or field that tells you if user created or switched over to a new tile page?

### DISCUSSION THREAD

**Discussion by pt-fasaldivar** [elgatosf/elgato#38] [Help]
Is there a dedicated SDK event or way of knowing when the user changes tile pages vs when he/she deletes a tile. Both actions seem to call the same WillDisappear event. Please let me know if this is possible. My ultimate goal is to differentiate if the user is switching tile pages or just decided to remove the tile from the StreamDeck device.

Thank you,

**Comment by ChekTek** [comment]
Hi!

There is not currently a way to differentiate between a key action leaving the display and a key action being deleted, but we have had this request multiple times and have already created a ticket internally that has been prioritized!
